test/local/account_routes.js

```diff
@@ -121,3 +122,34 @@ test(
     })
   }
 )
+
+test(
+  '/recovery_email/status logs query reason',
+  function (t) {
+    var mockLog = {
+      begin: function () {},
```

Comment:
You can use `mocks.mockLog({ increment: function() ... })` to get the definition of `begin` and some sinon spy stuff for free.